primitives/state-machine/src/testing.rs

```diff
@@ -48,20 +48,22 @@ pub struct TestExternalities<H: Hasher, N: ChangesTrieBlockNumber = u64>
 where
 	H::Out: codec::Codec + Ord,
 {
-	overlay: OverlayedChanges,
+	/// The overlay changed storage.
+	pub overlay: OverlayedChanges,
```

Comment:
Why do you make them public?

Reply:
https://github.com/paritytech/substrate/pull/8038/files#diff-a56434224c915b46e4750307b5b8ebd481757e2798119a6feea43e539f39d103R113

Reply:
Wrong link? :D I don't see the reason for this :D

Reply:
Hmm the link changed as you push/pull commits? strange, or I just did it wrong. 

And yes I am only using backend and extensions. Removed.